### PR TITLE
Error messages corrected

### LINKED ISSUE #1516: Two lines of error messages are reversed.
The error messages of lines 100 and lines 108 are reversed in cmd/katib-controller/v1beta1/main.go. The first error should be "Failed to create the manager." and the other should be "unable add APIs to scheme".

### PR DESCRIPTION
**Pull request by himanshu007-creator**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1516 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: cmd/katib-controller/v1beta1/main.go [hunk 1]
```diff
@@ -97,15 +97,15 @@ func main() {
...
+		log.Error(err, "Failed to create the manager")
 		os.Exit(1)
 	}
 
 	log.Info("Registering Components.")
 
 	// Setup Scheme for all resources
 	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
-		log.Error(err, "Fail to create the manager")
+		log.Error(err, "unable add APIs to scheme")
```
Comment (andreyvelich) [inline cmd/katib-controller/v1beta1/main.go:108]: Let's be consistent with the first capital letter.
```suggestion
		log.Error(err, "Unable to add APIs to scheme")
```

**Review by andreyvelich — COMMENTED** [review]
Thank you for taking this @himanshu007-creator and welcome to the community!
Just small comment from me.
/ok-to-test

**Review by gaocegege — APPROVED** [review]
/lgtm

**Pull request merged**